bbp/modules/default.nix

```diff
@@ -781,6 +781,17 @@ let
             conflicts = conflicts-modules;
         };
 
+        snakemake = pkgs.envModuleGen rec {
+            name = "snakemake";
+            moduleFilePrefix = "nix/py36";
+            isLibrary = true;
```

Comment:
What does 'isLibrary' stand for?
Do I need it here?

Reply:
isLibrary is useful for general binary C language libraries. It will automatically set tup environment variable to allow to link easily with this library. If your module is pure python and do not have a public C API. It is useless.

Reply:
OK, thanks for the clarification.